addons/mail/static/src/components/popover_view/popover_view.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="mail.PopoverView" owl="1">
         <t t-if="popoverView">
-            <div class="o_PopoverView" t-attf-class="{{ className }}" t-ref="root">
+            <div class="o_PopoverView bg-white border shadow-sm" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
@xlu-odoo, According to the SCSS guidelines (https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines), `border` should be before `background`.
Otherwise, the rest is fine for me :)